data/json/npcs/missiondef_robofac.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "id": "MISSION_ROBOFAC_INTERCOM_1",
+    "type": "mission_definition",
+    "name": "Return Field Data",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 5,
+    "item": "robofac_test_data",
+    "value": 1000,
+    "start": {
+      "assign_mission_target": { "om_terrain": "field", "reveal_radius": 2, "random": true, "search_range": 10 },
+      "update_mapgen": {
+        "rows": [
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "          M             ",
+          "            b b         ",
+          "         b b b@bb       ",
+          "             b b        ",
+          "                        ",
+          "                        "
+        ],
+        "terrain": { "@": "t_grass", "b": "t_dirt", "M": "t_dirt" },
+        "fields": { "b": { "field": "fd_blood", "density": 1 } },
+        "mapping": { "@": { "items": { "item": "produce" }, "item": [ { "item": "EMPbomb", "count": 2 }, { "item": "corpse" } ] } },
+        "monster": { "M": { "monster": "mon_robofac_prototype" } }
+      }
+    },
+    "end": {
+      "effect": [
+        { "npc_add_var": "npc_intercom_trade", "type": "dialogue", "context": "intercom", "value": "yes" },
+        { "u_add_var": "completed_robofac_intercom_1", "type": "dialogue", "context": "intercom", "value": "yes" }
+      ]
+    },
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "followup": "MISSION_FREE_MERCHANTS_EVAC_3",
+    "dialogue": {
+      "describe": "No I said ... [*You Hear a short, muffled conversation from across the intercom*] /n Well it seems we do have a use for you.  It's dangerous and you are likely to die, but if you complete it we will allow you limited access to our resources.",
+      "offer": "One of our scientists recently left the lab to perform a field test on a prototype robot, but failed to return, and has not been heard since.  Investigate the test and return with her and the prototype, or failing that return with the data recorded that was attached to our prototype.",
+      "accepted": "We appreciate your help, good luck.",
+      "rejected": "Don't expect our help then.",
+      "advice": "If the robot remains operational don’t try to fight it head on, because it WILL kill you.  Dr Prado left the with a handful of EMP grenades, use those to disable the robot.",
```

Comment:
```suggestion
      "advice": "If the robot remains operational don’t try to fight it head on, because it WILL kill you.  Dr. Prado left the Hub with a handful of EMP grenades, use those to disable the robot.",
```